doc/nrf/releases_and_maturity/releases/release-notes-2.6.0.rst

```diff
@@ -0,0 +1,1525 @@
+  * Support for nRF9151 DK on the majority of :ref:`cellular IoT samples <cellular_samples>`.
+  * The :ref:`serial_lte_modem` application can now be used to turn an nRF91 Series SiP into a standalone modem that can be used through Zephyr's cellular modem driver.
+
+* Security:
+
+  * Introduced the new :ref:`trusted_storage_readme` library which uses Hardware Unique Key to provide an implementation of PSA Certified Secure Storage API without the use of TF-M Platform Root of Trust (PRoT).
+
+* Other:
+
+  * New :ref:`networking_samples`: :ref:`net_coap_client_sample` and :ref:`http_server`.
+    These samples can be built for either a cellular IoT target or Wi-Fi target.
+
+Added the following features as experimental:
+
+* Wi-Fi:
+
+  * Raw Wi-Fi reception in both monitor and promiscuous modes.
+  * Software-enabled Access Point (SoftAP) with the primary use case being provisioning over Wi-Fi instead of the existing provisioning over Bluetooth LE solution.
+  * New samples: :ref:`wifi_softap_sample` and :ref:`wifi_monitor_sample`.
+
+* Sidewalk:
+
+  * Downlink File Transfer over Bluetooth LE, providing Sidewalk bulk transfer mode and integration of AWS IoT FUOTA (Firmware Updates over-the-air) service.
+
+* Thread:
+
+  * Thread commissioning over authenticated TLS (TCAT) to address the needs of professional installation and commercial building scenarios, where scanning physical install codes is an inadequate solution.
+* DFU:
+
+  * Possibility to split the application image between internal and external memory.
+    See the documentation on :ref:`qspi_xip` and :ref:`smp_svr_ext_xip`.
+
+* nRF Cloud:
+
+  * The :ref:`nrf_cloud_multi_service` sample now includes experimental support for the following:
+
+    * Runtime installation of TLS certificates using the TLS Credentials Shell when built for :ref:`Wi-Fi connectivity <nrf_cloud_multi_service_building_wifi_conn>` on an nRF5340 DK with an nRF7002 EK.
+    * Communication using either MQTT or CoAP over Wi-Fi.
+    * Better out-of-box experience by using the new nRF Cloud Security Services :ref:`auto-onboarding feature <nrf_cloud_multi_service_provisioning_service>`.
+      This makes it easier to connect a new device to nRF Cloud, including installation of security credentials.
+  * The :ref:`nrf_cloud_rest_device_message` sample also includes experimental support for the new nRF Cloud Security Services :ref:`auto-onboarding <remote_prov_auto_onboard>` feature.
+
+* Other:
+
+  * Link Time Optimization that brings flash usage improvements.
+    This has been tested on Matter over Wi-Fi template sample and it reduced flash by 73 KB (931 KB to 858 KB) for debug build and 64 KB (840 KB to 776 KB) on release build.
+
+Improved:
+
+* Bluetooth LE Audio:
+
+  * The application nRF5340 Audio is now called :ref:`nrf53_audio_app` and it has been refactored, expanded and improved, using Nordic Semiconductor's :ref:`softdevice_controller` instead of the previous dedicated LE Audio Controller subsystem.
```

Comment:
```suggestion
  * The application nRF5340 Audio is now called :ref:`nrf53_audio_app` and it has been refactored, expanded, and improved, using Nordic Semiconductor's :ref:`softdevice_controller` instead of the previous dedicated LE Audio Controller subsystem.
```